pyproject.toml

```diff
@@ -11,7 +11,15 @@ classifiers = [
     "Programming Language :: Python :: 3.11",
 ]
 description = "Demonstration of a tomography scan orchestrated by BlueSky"
-dependencies = [] # Add project dependencies here, e.g. ["click", "numpy"]
+dependencies = [
```

Comment:
Probably either update this dependency list - or drop it entirely because we are really only running the code in containers anyway.